illustrations.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<head>
+    <link rel="icon" href="src/images/早安.png" type="image/x-icon" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8" />
+    <title>RedcXca</title>
+    <script src="https://code.jquery.com/jquery-3.7.1.min.js"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+        crossorigin="anonymous"></script>
+    <link href="./src/css/illustrations.css" rel="stylesheet" />
+    <link href="./src/css/shared.css" rel="stylesheet" />
+    <style>
+        @import url("https://fonts.googleapis.com/css2?family=Dosis:wght@200..800&display=swap");
+    </style>
+</head>
+
+<body>
+    <div class="hero"></div>
```

Comment:
hero should probably go under navbar to not block it (or maybe its just the transparent-nav class)

Reply:
okay its just the transparent-nav class (i forgot why it was there) just take it out for now